x-pack/platform/plugins/shared/task_manager/server/saved_objects/model_versions/task_model_versions.ts

```diff
@@ -53,4 +53,27 @@ export const taskModelVersions: SavedObjectsModelVersionMap = {
       create: taskSchemaV3,
     },
   },
+  '4': {
```

Comment:
we should make sure to pull out these model version changes into a separate PR for intermediate release

Reply:
done: https://github.com/elastic/kibana/pull/215431